packages/sanity/src/core/form/store/types/nodes.ts

```diff
@@ -13,23 +13,34 @@ import {FormNodePresence} from '../../../presence'
 import {ArrayOfObjectsMember, ArrayOfPrimitivesMember, ObjectMember} from './members'
 import {FormFieldGroup} from './fieldGroup'
 
-/** @public */
+/**
+ * @public
```

Comment:
If I'm not mistaken this was one of the things that @bjoerge mentioned at one point he thought would be best to hide for now (I'm assuming this and the rest in this types though I'm not sure how realistic that is).

My only two cents is that if we're keeping it visible: we should add a small description :) 

@rexxars do you have thoughts on this, visible or hide?

Reply:
Bjørge has more context than me on this - if he feels we should hide it for now, I'd follow that advice - we can talk more about the way forward for this when he's back.

Reply:
I added `@hidden` but it does not seem to work when the tag is `@public`. Changing to `@beta` or `@internal` will fail the build since there are other public interfaces using this as a base. Should all of them be hidden for the time being?

Reply:
> I added `@hidden` but it does not seem to work when the tag is `@public`. 

In what layer does it not work? Build? Rendering? I think it would be ideal if we _could_ make it work, as some things are theoretically just "temporarily hidden until we can properly document it" or similar.

Reply:
@binoy14 are you sure? 🤔 I just re-ran the etl on this PR and looking at the rendering of the website none of these items are showing up / accessible.

Reply:
Sorry ignore me, I was checking on the wrong dataset 😬, made it hidden now!